index.html

```diff
@@ -5842,37 +5868,39 @@ <h3>Algorithm</h3>
           according to the steps in the <a href="#expansion-algorithm">Expansion algorithm</a>:</p>
 
         <ol>
-          <li>Create a new {{Promise}} <var>promise</var> and return it.
+          <li id="idl-expand-step-1">Create a new {{Promise}} <var>promise</var> and return it.
             The following steps are then deferred.</li>
-          <li class="changed">If the provided <a data-lt="jsonldprocessor-expand-input">input</a>
+          <li id="idl-expand-step-2" class="changed">If the provided <a data-lt="jsonldprocessor-expand-input">input</a>
             is a <a>RemoteDocument</a>,
             initialize <var>remote document</var> to <a data-lt="jsonldprocessor-expand-input">input</a>.</li>
-          <li>Otherwise, if the provided <a data-lt="jsonldprocessor-expand-input">input</a>
+          <li id="idl-expand-step-3">Otherwise, if the provided <a data-lt="jsonldprocessor-expand-input">input</a>
             is a <a>string</a> representing the <a>IRI</a> of a remote document, await and dereference it as <var>remote document</var>
             using <a>LoadDocumentCallback</a>, passing <a data-lt="jsonldprocessor-expand-input">input</a>
             for {{LoadDocumentCallback/url}},
             the {{JsonLdOptions/extractAllScripts}} option from <a data-lt="jsonldprocessor-expand-options">options</a>
             for <a data-link-for="LoadDocumentOptions">extractAllScripts</a>.</li>
-          <li class="changed">If {{RemoteDocument/document}}
-            from <var>remote document</var> is a <a>string</a>, transform into the <a>internal representation</a>.
+          <li id="idl-expand-step-4" class="changed">If {{RemoteDocument/document}}
+            from <var>remote document</var> is a <a>string</a>, transform into the <a>internal representation</a>
+            (see <a href="#internal-representation-in-implementations" class="sectionRef"></a>).
```

Comment:
```suggestion
            (see <a href="#serialization-and-deserialization" class="sectionRef"></a>).
```
seems more appropriate